### ISSUE TITLE
Custom pretrained resnet-50 model

### ISSUE THREAD

**Issue by lunix1982** [facebookresearch/deepmask#118]
Hello.
I have trained a custom resnet-50 model using the [Facebook implementation for Torch](https://github.com/facebook/fb.resnet.torch).
But the following error appears during the training of the Deep Mask model:
```SpatialZeroPadding.lua:87: attempt to index local 'cg_output' (a nil value)
stack traceback:
	.../lua/5.2/nn/SpatialZeroPadding.lua:87: in function 'updateGradInput'
	.../lua/5.2/nn/Module.lua:31: in function <.../lua/5.2/nn/Module.lua:29>
	[C]: in function 'xpcall'
	.../lua/5.2/nn/Container.lua:63: in function 'rethrowErrors'
	.../lua/5.2/nn/Sequential.lua:88: in function <.../lua/5.2/nn/Sequential.lua:78>
	[C]: in function 'xpcall'
	.../lua/5.2/nn/Container.lua:63: in function 'rethrowErrors'
	.../lua/5.2/nn/Sequential.lua:88: in function <.../lua/5.2/nn/Sequential.lua:78>
	[C]: in function 'xpcall'
	.../lua/5.2/nn/Container.lua:63: in function 'rethrowErrors'
	.../lua/5.2/nn/Sequential.lua:88: in function 'backward'
	.../deepmask/TrainerDeepMask.lua:88: in function 'train'
	train.lua:117: in main chunk
```
### resnet-50 settings
Data structure:
```
/data
    /label1
        img01.png
        img02.png
        ...
    /label2
    ...
```
Image resolution: ```256x256```
There are options I have tried to train the model:
```
- th main.lua -data data -depth 50 -nClasses 35
- th main.lua -data data -depth 50 -nClasses 35 -nThreads 4 -shareGradInput true
```

### Deep Mask settings
Images contain multiple objects the resnet-50 model was trained on.
Image resolution: ```1024x1024```
Coco annotations file contains polygons in format: ```x1, y1, x2, y2, ...```

1. Did anyone try to use a custom pretrained model or does know how to fix that?
2. If I use https://s3.amazonaws.com/deepmask/models/resnet-50.t7 I get messages like:
```[train] | epoch 00005 | s/batch 0.11 | loss: 0.09541 	
[train] | epoch 00006 | s/batch 0.11 | loss: 0.09705 	
[test]  | epoch 00006 | IoU: mean 000.00 median 000.00 suc@.5 000.00 suc@.7 000.00 | acc 095.26 | bestmodel x
```
Is that normal or there is something wrong with the custom data?
